### PR TITLE
HW2: Rebased onto main branch

### PR DESCRIPTION
**Pull request by cherieliu**
Added personal README to proposal file, created project files, and added the project repo link to the file

### CODE REVIEW

File: students/cl3945.md [hunk 1]
```diff
@@ -1,23 +1,15 @@
...
-UNI: cl3945
-
-Github: [link](https://github.com/cherieliu)
-
-Github: [link](https://github.com/cherieliu/cherieliu/blob/main/README.md)
 
 ## Project Proposal
-
-A tool that helps people choose their HTML colors without needing to find the RGB code. It will assist in color picking by following a certain color scheme, so that the frontend is more cohesive.
+- [color-template](./projects/javascript/color-template.md)
```
Comment (timkpaine) [inline students/cl3945.md:12]: update link path and re-squash 😈

File: students/cl3945.md [hunk 2]
```diff
@@ -1,23 +1,15 @@
...
 ## Project Contributions
-
-[Material UI](https://github.com/mui/material-ui/issues)
-
-[p5.js](https://github.com/processing/p5.js/issues)
-
-[React](https://github.com/facebook/react/issues)
-
-[numpy](https://github.com/numpy/numpy/issues)
+- [Material UI](./projects/javascript/material-ui.md)
```
Comment (timkpaine) [inline students/cl3945.md:15]: update link path and re-squash 😈

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: students/cl3945.md (hunk 1 above)
Comment (cherieliu) [reply students/cl3945.md]: Fixed in new commit!

File: students/cl3945.md (hunk 2 above)
Comment (cherieliu) [reply students/cl3945.md]: Fixed in new commit!

**Pull request merged**